Automattic/wp-calypso
# Add Akismet Business to upgrade mapping

## Proposed Changes

* Add upgrade mapping for Akismet Business from Akismet Pro (40k) product

## Testing Instructions

* Sandbox `public-api.wordpress.com` and enable store sandbox
* Purchase Akismet Pro: http://calypso.localhost:3000/checkout/akismet/ak_plus_yearly_1
* Go to your purchase in http://calypso.localhost:3000/me/purchases and find your Akismet Pro purchase
* The "Upgrade" button should direct you to: http://calypso.localhost:3000/checkout/akismet/ak_ent_yearly_1

## Pre-merge Checklist
